tcs/csp_solver_pc_Rankine_indirect_224.cpp

```diff
@@ -2286,3 +2277,32 @@ double C_pc_Rankine_indirect_224::Interpolate(int YT, int XT, double X, double Z
 
 } // Interpolate
 
+double C_pc_Rankine_indirect_224::Calculate_T_htf_cold_Converge_Cp(double q_dot_htf /*kWt*/, double T_htf_hot /*K*/, double m_dot_htf /*kg/s*/)
```

Comment:
Would this method make more sense in the HTF class so we can use it more broadly?

Reply:
Probably so... However, I don't know how generally robust this convergence method is and if it would be better to switch it to the monotonic equation solver...